### PR TITLE
feat: add HPA behaviour to troubleshooting guide

### PR DESCRIPTION
**Pull request by papanito**

### CODE REVIEW

**Comment by papanito** [comment]
@catherineluse some more addition to the troubleshooting guide

File: docs/troubleshooting.md [hunk 1]
```diff
@@ -142,3 +142,11 @@ Based on the above log, you can add the following entry to remove the operation:
 ```json
 {"op":"remove", "path":"/spec/template/spec/hostNetwork"}
 ```
+
+### My bundle which has a Horizontal Pod Autoscaler is in state `Modified`
```
Comment (jtravee) [inline docs/troubleshooting.md:146]:
```suggestion
### Bundle has a Horizontal Pod Autoscaler (HPA) in modified state
```

File: docs/troubleshooting.md [hunk 2]
```diff
@@ -142,3 +142,11 @@ Based on the above log, you can add the following entry to remove the operation:
...
+
+For bundles with an HPA is expected that it is in state `Modified`, as the bundle contains fields that differ from the state of the Bundle at deployment.
```
Comment (jtravee) [inline docs/troubleshooting.md:148]:
```suggestion
For bundles with an HPA, the expected state is `Modified`, as the bundle contains fields that differ from the state of the Bundle at deployment.
```

File: docs/troubleshooting.md [hunk 3]
```diff
@@ -142,3 +142,11 @@ Based on the above log, you can add the following entry to remove the operation:
...
+
+In such cases you have to define a patch in the `fleet.yaml` to ignore this field according to [bundle-diffs](./bundle-diffs.md).
```
Comment (jtravee) [inline docs/troubleshooting.md:150]:
```suggestion
You must define a patch in the `fleet.yaml` to ignore this field according to [bundle-diffs](./bundle-diffs.md).
```

Comment (jtravee) [reply docs/troubleshooting.md:150]: Please specify what field you're referring to when you say `ignore this field`. Ty!

File: docs/troubleshooting.md [hunk 4]
```diff
@@ -142,3 +142,11 @@ Based on the above log, you can add the following entry to remove the operation:
...
+
+An example of configuring this for the replicas field of a Deployment can be found [here](https://github.com/axeal/fleet-test/blob/320229/fleet.yaml).
```
Comment (jtravee) [inline docs/troubleshooting.md:152]:
```suggestion
Please refer [here](https://github.com/axeal/fleet-test/blob/320229/fleet.yaml) for an example of how to configure settings for the **replicas** field of a Deployment.
```

**Review by jtravee — CHANGES_REQUESTED** [review]

File: docs/troubleshooting.md (hunk 4 above)
Comment (papanito) [reply docs/troubleshooting.md]: Thanks for the review and the corrections. Please check my latest changes.

File: docs/troubleshooting.md [hunk 5]
```diff
@@ -169,6 +169,25 @@ Based on the above log, you can add the following entry to remove the operation:
 
 1. You can also force update the `gitrepo` to perform a manual resync. Select **GitRepo** on the left navigation bar, then select **Force Update**.
 
+### Bundle has a Horizontal Pod Autoscaler (HPA) in modified state
+
+For bundles with an HPA, the expected state is `Modified`, as the bundle contains fields that differ from the state of the Bundle at deployment - usually `ReplicaSet`.
+
+You must define a patch in the `fleet.yaml` to ignore this field according to [`GitRepo` or `Bundle` stuck in modified state](#gitrepo-or-bundle-stuck-in-modified-state).
+
+Here an example of such a patch for the deployment `nginx` in namespace `default`
```
Comment (jtravee) [inline docs/troubleshooting.md:178]:
```suggestion
Here is an example of such a patch for the deployment `nginx` in namespace `default`:
```

**Review by jtravee — CHANGES_REQUESTED** [review]

File: docs/troubleshooting.md (hunk 5 above)
Comment (jtravee) [reply docs/troubleshooting.md:178]: Thanks! Just a minor correction on this line.

**Review by jtravee — APPROVED** [review]

**Review by StrongMonkey — APPROVED** [review]

**Comment by papanito** [comment]
Thanks a lot

**Pull request merged**